### DISCUSSION TITLE
ping: socket: Permission denied

### DISCUSSION THREAD

**Discussion by Tuharsky** [ltsp/ltsp#696] [General]
Hi,
I have encountered a problem. At LTSP client, logged in as regular user, opened terminal. Trying to ping something, but I get error

ping: socket: Permission denied

There is no problem doing this while logged at server directly.

**Comment by alkisg** [comment]
How are you creating your image, e.g. chrootless or chroot or VM?
`ping` works fine here in all cases.
Have you messed with any of the Linux capabilities?
https://unix.stackexchange.com/questions/617927/why-ping-works-without-capability-and-setuid

**Comment by map7** [comment]
I've had to allow permissions on ping before when I've had this issue.

    sudo setcap cap_net_raw=ep $(which ping)
